run/src/adapt.ts

```diff
@@ -4,12 +4,12 @@ export interface AdaptStream {
   (s: Stream<any>): any;
 }
 
-let adaptStream: AdaptStream = x => x;
+(global as any).adaptStream = (x => x) as AdaptStream;
```

Comment:
We should be careful with the `global` keyword: https://github.com/tc39/proposal-global see also http://kangax.github.io/compat-table/esnext/#test-global

Instead, let's do something like this https://github.com/ReactiveX/rxjs/blob/c7c337e134605c9629ccfcaaf7ef562cd2bb9c18/src/Rx.global.js